src/coffee/directives/api/models/parent/circle-parent-model.coffee

```diff
@@ -26,7 +26,7 @@ angular.module('uiGmapgoogle-maps.directives.api.models.parent')
 
       @setMyOptions = (newVals, oldVals) =>
         return if scope.settingFromDirective
-        unless _.isEqual newVals,oldVals
+        unless _.isEqual newVals,oldVals and newVals == oldVals and (newVals? and oldVals? and newVals.coordinates == oldVals.coordinates)
```

Comment:
To be safe here `_.isEqual newVals,oldVal` should have `()` as the parser somtimes dudges this `_.isEqual(newVals,oldVal)`.